Cargo.toml

```diff
@@ -6,15 +6,17 @@ license = 'Apache-2.0'
 name = "hydra-dx-math"
 description = "A collection of utilities to make performing liquidity pool calculations more convenient."
 repository = 'https://github.com/galacticcouncil/hydradx-math'
-version = "4.5.1"
+version = "4.6.0"
 
 [dependencies]
 primitive-types = {default-features = false, version = '0.8.0'}
 num-traits = {default-features = false, version = "0.2.14"}
-fixed = "1.9.0"
+fixed = "=1.15.0"
```

Comment:
is this `=` necessary?

Reply:
yes.otherwise it updated to 1.16 which does not work .